### DISCUSSION TITLE
Call for the new website

### DISCUSSION THREAD

**Discussion by Hans5958** [ScratchAddons/ScratchAddons#7142] [Discussion]
Have fresh ideas on the new website? You can post your ideas, concepts, proposal, etc here, and we can have a discussion.

The repositories as scope of the website is as follows.

1. https://github.com/ScratchAddons/website-v2
2. https://github.com/ScratchAddons/website-v2-script
3. https://github.com/ScratchAddons/website-i18n

This is made as few people have interests on creating the new website, and I welcome these as the main maintainer of [the current website](https://github.com/ScratchAddons/website-v2). This discussion is made here so that it is not restrained to [the website-v2 repository](https://github.com/ScratchAddons/website-v2).

Note that maintenance by me is not guranteed, and it depends on the implementation. The worst case scenario is that you will have a level of maintenance responsibility of the new site.

**Comment by Hans5958** [comment]
Few considerations of mine

- Consider all (open) [issues](https://github.com/ScratchAddons/website-v2/issues) and (pending) [PRs](https://github.com/ScratchAddons/website-v2/pulls) on the repository.
- Keep it as a static site as much as possible. JS frameworks are a good thing, but use it wisely; as needed (see ScratchAddons/website-v2#271)
    - I don't like having it as an (React/Vue/Svelte/etc) app, and no way we are doing SSR with such budgets.
- Consider whether separating the repository into multiple repositories, for docs, blog, etc. i18n should be kept separate for size reasons, and the website should be loadable when there is only English
- Please see context on the community Discord server, located on https://discord.com/channels/806602307750985799/806602307750985803/1203742096436101250

**Comment by Hans5958** [comment]
For the note, GrahamSH-LLK considers using Docusaurus, at least for the docs, as mentioned on ScratchAddons/website-v2#283.

**Reply by BlinzTheCoder** [reply]
The only framework that I think would not cause complications in the future is docusauras so I also support it

**Comment by Hans5958** [comment]
As another note (again), my personal opinion goes towards [Astro](https://astro.build), as an static site generator that offers good support of JS frameworks (React, Vue, Svelte, etc). I have used it for a while, so I might be biased, but believe it's pretty good.

**Comment by Secret-chest** [comment]
-1 for Astro, let's keep JS for web apps, not for replacing semantic HTML structure.

**Reply by Hans5958** [reply]
What do you mean? No one says that it replaces "semantic HTML structure." Like, do you even know what you are talking about?

**Reply by Secret-chest** [reply]
JavaScript was made for web interactivity, not content management.

**Reply by Hans5958** [reply]
...and who says that it is made for content management?
